tests/foreman/api/test_convert2rhel.py

```diff
@@ -261,8 +261,18 @@ def test_convert2rhel_oracle_with_pre_conversion_template_check(
     """
     major = version.split('.')[0]
     assert oracle.execute('yum -y update').status == 0
+    if oracle.execute('needs-restarting -r').status == 1:
+        oracle.power_control(state='reboot')
```

Comment:
Since we already have an reboot step in oracle setup fixture, we're just registering a system after that reboot, so I'd suggest to avoid rebooting twice and just try like register -> dnf update -> reboot